spec/unit/enforcement_spec.rb

```diff
@@ -1,15 +1,29 @@
 # frozen_string_literal: true
+class MyController # < ApplicationController
```

Comment:
This class should probably be destroyed after running the tests, otherwise you could have collisions with other tests.